Updated the Readme Doc

Devbysteph opened the pull request:
I improved the Readme doc, please review it.

Devbysteph committed:
improved the project readme docs

Devbysteph committed:
Update README.md

Devbysteph committed:
Update README.md

Devbysteph committed:
Added anony monkey emoji

Devbysteph committed:
Merge branch 'main' of https://github.com/Devbysteph/anon-msg-app

Devbysteph committed:
Merge branch 'vicradon:main' into main

Devbysteph committed:
Update README.md

vicradon commented on README.md:
```diff
@@ -1,34 +1,29 @@
+>Access all of AnonMsg🙈 features for free without paying a penny.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+## Download
```
Might want to replace this download with "View". You know it's a web app.

Devbysteph replied:
Okayy sure

vicradon commented on README.md:
```diff
@@ -1,34 +1,29 @@
+## Download
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+## App Screenshots
```
I think we can add this now. Both the desktop and mobile version.

Devbysteph replied:
Alright, noted. Is it responsive now?

vicradon commented on README.md:
```diff
@@ -1,34 +1,29 @@
+1. Anonymity 👓
+>When you send someone a secret message, our platform protects your privacy to keep you anonymous. You remain anonymous unless you decide to expose your true identity.
+2. Simple to Use 💫
+>We are continually improving the platform, to make AnonMsg🙈 as user-friendly as possible.To get started right now, all you need to do is to go [AnonMsg🙈](https://anon-msg-app.vercel.app/)and enter your login and password. 
```
Might want to replace the phrase "end to end" encryption with: "aes cipher".

Devbysteph replied:
Okay, Roger that.

vicradon commented on README.md:
```diff
@@ -1,34 +1,29 @@
+# AnonMsg🙈
 
-## Getting Started
+<img src="./public/Emoji.png" style: width="100px">
```
This image is broken... I think it has to do with the relative link. Try removing the `./public`.

Devbysteph replied:
Alright, the saved the image on the public folder that was why I referenced it. What do you think. I will look into you ASAP.

Devbysteph committed:
Updates the ReadMe

Devbysteph committed:
Add files via upload

Devbysteph committed:
Delete Emoji.png

Devbysteph committed:
Delete Logo placeholder.jpg

Devbysteph committed:
Add files via upload

Devbysteph committed:
added webapp screenshot & improve contents

Devbysteph committed:
Merge branch 'vicradon:main' into main

vicradon commented on README.md:
```diff
@@ -1,34 +1,37 @@
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+# AnonMsg🙈
+![Logoplaceholder](https://github.com/Devbysteph/anon-msg-app/blob/main/public/Logo%20placeholder.png) 
```
Can you also structure this image like the others? If it references your repo, it might not show if your repo is moved.

Devbysteph replied:
I've made modifications to them, thank you for pointing it out.

Devbysteph committed:
Uploaded screenshot images

Devbysteph committed:
Modified Image upload

I modified the images based on the correction, to ensure it referenced when marged.

Devbysteph committed:
Merge branch 'vicradon:main' into main

Devbysteph committed:
Modified readme

vicradon commented:
Thanksyou @Devbysteph for this contribution.
